src/api/common/utils/StringUtils.js

```diff
@@ -106,4 +106,20 @@ export function byteLength(str: ?string): number {
 		if (code >= 0xDC00 && code <= 0xDFFF) i--; //trail surrogate
 	}
 	return s;
+}
+
+/**
+ * replace all instances of substr in str with replacement
+ */
+export function replaceAll(str: string, substr: string, replacement: string): string {
```

Comment:
Should we just take replaceAll polyfill from core-js? We wouldn't need tests and we wouldn't need to migrate to builtin later

Reply:
Seems like we don't use core-js anymore so eh